holopy/scattering/scatterer/scatterer.py

```diff
@@ -99,7 +99,7 @@ def index_at(self, points, background=0):
         index = np.ones_like(domains, dtype=dtype) * background
         for i, n in enumerate(ns):
             index[domains==i+1] = n
-        return index
+        return xr.DataArray(index, attrs = {'_image_scaling':[0,self.n.max()], 'voxel':1})
```

Comment:
This is a bit of a missed opportunity to take full advantage of xarray by keeping track of the coordinates of each voxel. It should be pretty straightforward to pass in coords and dims according to the [xarray documentation](http://xarray.pydata.org/en/stable/data-structures.html#creating-a-dataarray). Let me know if you want help implementing this, but I think we should be able get everything working and merged in without it if you prefer.